vta/src/tsim/tsim_driver.cc

```diff
@@ -143,27 +133,15 @@ class Device {
   }
 
   void Launch(vta_phy_addr_t insn_phy_addr,
-              vta_phy_addr_t uop_phy_addr,
-              vta_phy_addr_t inp_phy_addr,
-              vta_phy_addr_t wgt_phy_addr,
-              vta_phy_addr_t acc_phy_addr,
-              vta_phy_addr_t out_phy_addr,
               uint32_t insn_count,
               uint32_t wait_cycles) {
-    dpi_->WriteReg(0x04, 0);
     dpi_->WriteReg(0x08, insn_count);
     dpi_->WriteReg(0x0c, insn_phy_addr);
-    dpi_->WriteReg(0x10, insn_phy_addr >> 32);
+    dpi_->WriteReg(0x10, 0);
     dpi_->WriteReg(0x14, 0);
-    dpi_->WriteReg(0x18, uop_phy_addr >> 32);
+    dpi_->WriteReg(0x18, 0);
```

Comment:
Why do we still have to zero out the `uop` address?

Reply:
The `uop` address as well as others `inp`, `wgt`, `acc`, and `out` will come on the instruction since the field is 32-bit. Like [here](https://github.com/dmlc/tvm/blob/master/vta/src/pynq/pynq_driver.cc#L117-L121)